Text/Megaparsec/Char/Lexer.hs

```diff
@@ -91,6 +91,9 @@ import qualified Text.Megaparsec.Char as C
 -- 'skipBlockComment' or 'skipBlockCommentNested' if you don't need anything
 -- special.
 --
+-- If you don't have a parser for a kind of comment,
```

Comment:
Maybe it's worth changing `use 'empty' as placeholder` to `pass 'empty'` or `use 'empty'`. It's nitpicking, but `empty` in this case is not a placeholder, it's just a parser that instantly fails and lets other things run instead.